### ISSUE TITLE
ISE Network Device Group API Wrong

### ISSUE THREAD

**Issue by maconrad** [CiscoISE/ciscoisesdk#38]
**Prerequisites**
* [x ] Have you tested the operation in the API directly?
* [x ] Do you have the latest SDK version?

**Describe the bug**
NetworkDeviceGroup.create uses wrong params in ISE 3.2. It uses othername but the API expects ndgtype (see screenshot below). This was probably changed/updated at some point.

https://ciscoisesdk.readthedocs.io/en/latest/_modules/ciscoisesdk/api/v3_1_0/network_device_group.html#NetworkDeviceGroup.create

**Expected behavior**
Update SDK to use correct name as seen in screeshot below. New: ndgtype, old: othername

**Screenshots**

<img width="1266" alt="image" src="https://user-images.githubusercontent.com/11851898/200886267-b95c8243-0830-42a6-a1aa-9d1e46dbfc27.png">

<img width="799" alt="image" src="https://user-images.githubusercontent.com/11851898/200886890-e1a115ce-690b-4293-bd63-9ae8cf48f129.png">

<img width="913" alt="image" src="https://user-images.githubusercontent.com/11851898/200887051-a115f1ec-9141-46c4-bc9a-f11388c66b21.png">

**Environment (please complete the following information):**
* ISE version and patch: 3.2
* Python version: 3.10.7
* SDK version: 2.0.8
* OS version: OSX

**Comment by bvargasre** [comment]
Hello @maconrad, currently version 3.2 is not supported, when we receive the corresponding documentation we will release a new version that supports it and you will be notified.

**Comment by boomsfib** [comment]
I'm also encountering this issue on 3.1 with Patch 4.  I was on 3.1 (w/ no Patches) and I was able to create a Network Device Group with othername, but I was not able to delete the Group.  I kept getting a CRUD operation exception.  I applied Patch 4, which resolved the issue of not being able to delete Network Device Groups, but now I'm unable to create them using param othername and using ndgtype as a param throws up an error. 

I did discover a work around however to create by wrapping everything under the payload param.

So rather than this
`api.network_device_group.create(name='Location#All Locations#TestSite')`

I did this instead
`api.network_device_group.create(payload= {'NetworkDeviceGroup': {'name': 'Location#All Locations#TestSite', 'ndgtype': 'Location'}})`
